go.mod

```diff
@@ -339,3 +339,5 @@ require (
 	k8s.io/klog v1.0.0 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace golang.org/x/crypto => golang.org/x/crypto v0.31.0 // cve ?
```

Comment:
// cve ?

Reply:
oh ty, forgot to add the number of cve there